File: src/Guias/TP5/Ej3/P2/MovablePoint.java
package Guias.TP5.Ej3.P2;

public class MovablePoint extends Point implements Movable{
    MovablePoint(double x, double y){
        super(x,y);
    }

    @Override
    public void moveEast(double delta) {
        x+=delta;
    }

    @Override
    public void moveNorth(double delta) {
        y+=delta;
    }

    @Override
    public void moveSouth(double delta) {
        y-=delta;
    }

    @Override
    public void moveWest(double delta) {
        x-=delta;
    }
}

File: Ruby/Guias/G9/Ej2/movable_point.rb
require_relative 'movable'
require_relative 'point'
class MovablePoint < Point
  include Movable

  def move_down(_delta)
    @y-=_delta
  end
  def move_up(_delta)
    @y+=_delta
  end
  def move_left(_delta)
    @x-=_delta
  end
  def move_right(_delta)
    @x+=_delta
  end
end